project_4/index.html

```diff
@@ -0,0 +1,384 @@
+                <h2 data-aos="fade-right">What is Our Mission?</h2>
+                <p class="mission-text">The Sustainable By Design Conference aims to define sustainable design,
+                    understanding that
+                    sustainability pertains to the environment, social welfare, and economic welfare. We will reevaluate
+                    current design practices in the context of our ourselves, our communities, and the natural world. We
+                    hope to address pressing issues related to the way in which our world is currently heading and offer
+                    practical design solutions and goals to change our path towards a more sustainable one. This
+                    conference will also allow us to open up a dialogue and work together to find better solutions in
+                    making the design world, and the rest of the world, more sustainable. </p>
+            </div>
+            <div class="right-column">
+                <h2 data-aos="fade-left"> Who should come?</h2>
+                <p class="mission-text">Of course, anyone is welcome to come and learn sustainabile design if that if
+                    your passion. We
+                    believe every designer should be informed about sustainability, and the cost of our actions if we
+                    continue designing in the way we design now. Sustainanbility should be on every designer's mind
+                    before they begin a project whether you are a graphic designer, fashion designer, or a product
+                    designer; it's all
+                    relevant. It is also important to hear from a variety of practicers and perspectives from every
+                    field, because believe it or not, a lot of fields of design are interconnected; it is important
+                    to learn about their practices moving forward in the age of our ever changing climate and
+                    economy.</p>
+            </div>
+
+            <div class="quotes">
+                <h2 data-aos="fade-right">What Are People Saying?</h2>
+                <blockquote class="quote">"From top to bottom the speakers and topics were carefully curated. The
+                    execution was flawlessly organized. I made so many connections. Professional development has never
+                    been so fun."</blockquote>
+                <!-- https://revolveconference.com/ -->
+                <p> - Douglas Davis, Creative Strategist - </p>
+                <button type="button" class="button-reset button-next">Next</button>
+            </div>
+        </section>
+
+        <!-- SPEAKERS -->
+        <h2 class="section-header">Speakers</h2>
+        <section id="speakers" class="speakers">
+
+            <ul class="speaker-info" data-aos="fade-up" data-aos-anchor-placement="top-center">
+                <a href="https://www.linkedin.com/in/maxinebedat" title="Link to Maxine Bedat's LinkedIn page">
```

Comment:
The _direct_ child of an `ul` or `ol` element _must_ be an `li`, so this needs to be wrapped in that element:
```html
                <li><a href="https://www.linkedin.com/in/maxinebedat"><img></a></li>
```
The `title` for your link here is a nice touch, btw! 👍🏻

Reply:
Actually, looking at the full markup now, I'd say this isn't really a list. The speakers themselves are a list, but the speaker details seems like they're info _for_ that list. I'd recommend updating your markup to something like this:
```html
            <ul class="speakers-list">
                <li class="speaker-info" data-aos="fade-up" data-aos-anchor-placement="top-center">
                    <a href="https://www.linkedin.com/in/maxinebedat" title="Link to Maxine Bedat's LinkedIn page">
                        <img src="images/Maxine_Bedat.jpg" alt="">
                    </a>
                    <!-- left the alt text blank for the headshot images because they do not provide new or helpful information--the name and info are the most important-->
                    <h3 class="speaker-name">Maxine Bedat</h3>
                    <p class="speaker-title">Executive Director at New Standard Institute</p>
                    <p class="speaker-paragraph">Founder and director of the New Standard Institute. She co-founded the fashion brand, Zady, focusing on sustainable practices with the best materials, low environmental footprints, fair labor, and style. Zady was named of the world’s “Most Innovative Companies” in retail.</p>
                </li>
                <li>...Next speaker details...</li>
            </ul>
```